File: ws/morillo/u2/ws24 Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Shark.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author David Morillo, JavaSquad, DCCO-ESPE
 */
public class Shark extends Fish{

    public Shark(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    @Override
    public void Feed(int food) {
        System.out.println("Feeding the Shark with " + food + " seal");
    }
    
}

File: hw/villagomez/u2/hw18ZooJS/ec.edu.espe.zoo.model/Shark.js
const Fish = require('./Fish');
const Cage = require('./Cage'); 

class Shark extends Fish {

    constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    feed(food) {
        console.log("Feeding to Shark");
    }
    
}

module.exports = Shark; 